embedding-calculator/src/docs/find_faces_base64_post.yml

```diff
@@ -0,0 +1,91 @@
+tags:
+  - Core
+summary: 'Find faces in the given image in BASE64 format and return their bounding boxes.'
+description: 'Returns bounding boxes of detected faces on the image.'
+operationId: findFacesBase64Post
+consumes:
+  - application/json
+produces:
+  - application/json
+parameters:
+  - in: body
+    name: file
+    type: file
+    required: 'true'
+    description: 'A picture with at least one face in BASE64 format.'
+  - in: query
+    name: limit
+    description: 'The limit of faces that you want recognized. Value of 0 represents no limit.'
+    type: integer
+    default: 0
+  - in: formData
+    name: det_prob_threshold
+    description: 'The minimum required confidence that a found face is actually a face. Decrease this value if faces are not detected. Valid values are in the range (0;1).'
+    type: float
+  - in: formData
```

Comment:
This is also `query`, the same in `find_faces_post.yml`

Reply:
Fixed